x-pack/plugins/security_solution/public/management/pages/policy/view/ingest_manager_integration/endpoint_policy_create_extension/endpoint_policy_create_extension.tsx

```diff
@@ -67,12 +80,21 @@ const HelpTextWithPadding = styled.div`
  */
 export const EndpointPolicyCreateExtension = memo<PackagePolicyCreateExtensionComponentProps>(
   ({ newPolicy, onChange }) => {
+    const isPlatinumPlus = useLicense().isPlatinumPlus();
+    const isEnterprise = useLicense().isEnterprise();
+
     // / Endpoint Radio Options (NGAV and EDRs)
     const [endpointPreset, setEndpointPreset] = useState<EndpointPreset>('NGAV');
     const [selectedCloudEvent, setSelectedCloudEvent] = useState<CloudEvent>('ALL_EVENTS');
     const [selectedEnvironment, setSelectedEnvironment] = useState<Environment>('endpoint');
     const initialRender = useRef(true);
 
+    // Show NGAV license note when Gold and below
+    // Show other licenses note when Platinum and Below
+    const showNote =
```

Comment:
Are these checks enforced via the API (the fleet Create package policy server extension)?

Reply:
Oh, not necessary since it's only about showing a note of what capabilities are already enabled with the current license level.

In practice, the preset options only change the events collection of the policy, and we let the policy factory handle the protections.